lib/process-services/src/lib/form/widgets/people/people.widget.spec.ts

```diff
@@ -138,13 +191,63 @@ describe('PeopleWidgetComponent', () => {
             email: 'john@test.com'
         };
         widget.ngOnInit();
-
-        const involvedUser = fixture.debugElement.nativeElement.querySelector('input[data-automation-id="adf-people-search-input"]');
-
         fixture.detectChanges();
         await fixture.whenStable();
 
-        expect(involvedUser.value).toBe('John Doe');
+        const chip = await loader.getHarness(MatChipHarness.with({ selector: '[data-automation-id="adf-people-widget-chip-people-id"]' }));
```

Comment:
This line is repeated couple of times, maybe we can move it to a function instead?

Reply:
done, ty